validation/userValidation.js

```diff
@@ -1,15 +1,21 @@
-const Joi = require("@hapi/joi");
+const Joi = require('@hapi/joi');
 
 const registrationSchema = Joi.object({
   username: Joi.string().min(3).max(30).required(),
   email: Joi.string().email().required(),
   password: Joi.string()
     .pattern(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/)
```

Comment:
Deluje mi da ti pattern za proveru sifre nije dobar tj. da mozes da imas sifru koja sadrzi samo slova i brojeve. Mislim da bi trebalo omoguciti da se koriste svi znakovi za sifru uz to da mora postojati slovo i broj u sifri.

Reply:
U pravu si, kada posaljem "pass-word1", ne matchuje, da li imas predlog kako bolje da napisemo regex?

Reply:
`/^(?=.*[A-Za-z])(?=.*\d).{8,}$/` probaj ovako, da umesto slova i brojeva matchujes sve karaktere.

Reply:
Super, ovako radi :+1: